ci-operator/step-registry/rosa/hypershift-operator/deploy-pr/rosa-hypershift-operator-deploy-pr-commands.sh

```
@@ -0,0 +1,173 @@
+set -o nounset
+set -o errexit
+set -o pipefail
+
+LOCK_ACQUIRED=""
+MC_KUBECONFIG=""
+
+cleanup() {
+  local children
+  children=$(jobs -p) || true
+  if [[ -n "${children}" ]]; then
+    kill ${children} && wait
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)